wordpress-mobile/WordPress-Android
# Adds the is_beta parameter to the fetch block layouts call

Depends on https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1855

**Description**
This PR adds `type` and `is_beta` parameters in fetch block layouts call both for WPCom and self hosted site.
This make it so that the api will be able to return type ( mobile or web ) specific page layouts as well as beta versions.

**To test**
1. Run the app in `debug` mode
2. Press the fab icon at the bottom right of the *My Site* screen
3. Select *Site page*
4. **Verify** that layouts are fetched

**Context**
This PR clones the [iOS implementation](https://github.com/wordpress-mobile/WordPress-iOS/pull/15678)
